src/main/java/uk/gov/hmcts/reform/managecase/client/datastore/ChangeOrganisationRequest.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.hmcts.reform.managecase.client.datastore;
```

Comment:
Should this be in one of the model packages: e.g. `uk.gov.hmcts.reform.managecase.domain` alongside org policy etc..

Reply:
I was suggesting this get moved out of the Client DataStore area and into MCA's own domain package:  `uk.gov.hmcts.reform.managecase.domain`.

Reply:
ah right, understood

Reply:
👍